server.js

```diff
@@ -1,31 +1,90 @@
+'use strict';
+
+// TODO: switch from 16-day to 8 day forecast
 // ===== packages ===== //
 
 const express = require('express');
 require ('dotenv').config();
 const cors = require('cors');
+
 const superagent = require('superagent');
 const { response, json } = require('express');
 
+const pg = require('pg');
+
 // ===== global variables ===== //
 
 const PORT = process.env.PORT || 3003;
 const locationApiKey = process.env.GEOCODE_API_KEY;
 const weatherApiKey = process.env.WEATHER_API_KEY;
 const trailsApiKey = process.env.TRAIL_API_KEY;
+const databaseUrl = process.env.DATABASE_URL;
+
+
 const app = express();
 app.use(cors());
 
+const client = new pg.Client(databaseUrl);
+client.on('error',(error) => console.error(error));
+
 // ===== routes ===== //
 
-function sendLocation (request,response){
+function checkSql (request,response) {
+
+  const userSearch = request.query.city;
+  const queryCheck = 'SELECT * FROM user_search;';
+
+  client.query(queryCheck)
+
+    .then(fromSql => {
+
+      let counter = 0;
+      for (let value of fromSql.rows){
+
+        if (value.search_query === userSearch){
+          console.log('PULLED FROM SERVER: ', value);
+          response.send(value);
+          break;
+
+        } else if (counter === fromSql.rows.length - 1){
+          sendLocationToApi(request,response);
+        }
+        counter++;
+      }
+    })
+}
```

Comment:
The set-up checking your database is structured very close to mine. The spaces between each piece of code actually really helps me read your code.